runtime/log.go

```diff
@@ -0,0 +1,26 @@
+package runtime
+
+import (
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+)
+
+// Create a logger that can be passed around through the environment.
```

Comment:
is there something I should enable for it to warn me about things like this?  'make test' nor running 'go vet' directly has indicating any problems.